tests/unit/bash/bash_pkg_conditional_rpm.bats.jinja

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Mock the "rpm" command
+# mocked package: coconut-4.5.17-8
+# The mock is used to avoid the need to have a specific RPM package in a
+# specific version installed on the system where this test is run.
+function rpm()
+{
+    if [[ "$*" == "--quiet -q coconut" ]] ; then
+        return 0
+    elif [[ "$*" ==  "-q --queryformat %{VERSION}-%{RELEASE} coconut" ]] ; then
+        echo "4.5.17-8"
+        return 0
+    else
+        echo "BATS mock for rpm doesn't support this command"
+        return 1
+    fi
+}
+export -f rpm
+
+@test "bash_pkg_conditional_rpm - test package presence" {
+    {{{ bash_pkg_conditional_rpm("coconut") | indent(4) }}}
+}
+
+@test "bash_pkg_conditional_rpm - test package version" {
+    {{{ bash_pkg_conditional_rpm("coconut", "5", "<") | indent(4) }}}
```

Comment:
Please consider introducing a macro that is rpm-agnostic, and that would be called as
`{{{ bash_pkg_conditional_compare("4.5.17-8", "5", "<") | indent(4) }}}`